### PR TITLE
Info

### LINKED ISSUE #19: Create Pull Request Template

### LINKED ISSUE #20: Add project prioritization document

### PR DESCRIPTION
**Pull request by Gvieve**
### What?
 * new `card_prioritization.md` to detail order of project issues
 * new PR template

### Why?
 * Housekeeping items

### Testing?
 * N/A

Closes #20 
closes #19

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### What?
+ *
+
+### Why?
+ *
+
+### How?
+ *
+
```
Comment (harrison-blake) [inline pull_request_template.md:9]: Is this necessary?

**Review by harrison-blake — CHANGES_REQUESTED** [review]
I added some notes!

File: pull_request_template.md (hunk 1 above)
Comment (Gvieve) [reply pull_request_template.md:9]: I think we should keep it

**Review by harrison-blake — APPROVED** [review]

**Comment by harrison-blake** [comment]
looks good!

**Pull request merged**